engine/src/main/resources/assets/i18n/menu_fa.lang

```diff
@@ -0,0 +1,244 @@
+{
+    "activate-module": "فعال کردن",
+    "add-server": "جدید اضافه کردن",
+    "add-server-tip": "شما می توانید با فشار دکمه ی اینتر اسم را در ادرس مورد نظر کپی کنید.",
+    "advanced-usage": "کاربرد پیشرفته",
+    "animate-grass": "خلق علفی کاملا طبیعی و زنده",
+    "animate-water": "خلق اب طبیعی",
+    "audio-settings": "تنظیمات صدا",
+    "audio-settings-title": "عنوان تنظیمات صدا",
+    "back": "بازگشت",
+    "binding-attack": "حمله",
+    "binding-autoMove-mode": "کلید خود جرکت",
+    "binding-backwards": "به عقب",
+    "binding-behavior-editor": "دستور تنظیم ویرایشگر",
+    "binding-chat": "کلید چت",
+    "binding-console": "کلید فرمان",
+    "binding-crouch": "قوز کردن / پایین رفتن",
+    "binding-crouch-mode": "حالت نشسته",
+    "binding-decrease-view-distance": "کاهش فاصله ی دید",
+    "binding-forwards": "رو به جلو",
+    "binding-frob": "هدف استفاده",
+    "binding-hide-hud": "اچ یو دی مخفی",
+    "binding-increase-view-distance": "افزایش فاصله ی دید",
+    "binding-jump": "پرش",
+    "binding-left": "گردش از چپ",
+    "binding-nui-editor": "NUI Editor",
```

Comment:
English string here - can you wrap this up just so the language is 100% finished? [1]